canonical/ubuntu-advantage-client
# enable: correct messaging for beta service

## Proposed Commit Message


> enable: correct messaging for beta service
>
> Previously, if the user tried to enable a beta service
> that was already enabled without using --beta, they
> would get incorrect messaging. Now they are correctly
> told that the service is already enabled.
> 
> Fixes: #1588

## Test Steps

Enable a beta service such as esm-apps
Then run `ua enable esm-apps` without the `--beta` flag.
Verify that the output correctly states that esm-apps is already enabled.

## Desired commit type::


## Checklist:
